src/sentry/incidents/subscription_processor.py

```diff
@@ -205,18 +211,41 @@ def process_update(self, subscription_update):
                     "result": subscription_update,
                 },
             )
-        aggregation_value = list(subscription_update["values"]["data"][0].values())[0]
 
         is_sessions_dataset = Dataset(self.subscription.snuba_query.dataset) == Dataset.Sessions
         if is_sessions_dataset:
+            aggregation_value = subscription_update["values"]["data"][0][
+                CRASH_RATE_ALERT_AGGREGATE_ALIAS
+            ]
             if aggregation_value is None:
                 metrics.incr("incidents.alert_rules.ignore_update_no_session_data")
                 return
+
+            try:
+                total_count = subscription_update["values"]["data"][0][
+                    CRASH_RATE_ALERT_SESSION_COUNT_ALIAS
+                ]
+                if CRASH_RATE_ALERT_MINIMUM_THRESHOLD is not None:
+                    min_threshold = int(CRASH_RATE_ALERT_MINIMUM_THRESHOLD)
+                    if total_count < min_threshold:
+                        metrics.incr(
+                            "incidents.alert_rules.ignore_update_count_lower_than_min_threshold"
+                        )
+                        return
+            except KeyError:
+                # If for whatever reason total session count was not sent in the update,
+                # ignore the minimum threshold comparison and continue along with processing the
+                # update. However, this should not happen.
+                logger.error(
+                    "Received an update for a crash rate alert subscription, but no total "
+                    "sessions count was sent"
+                )
```

Comment:
Might be worth making this `logger.exception` so that we get the details of the exception as well.